target/linux/realtek/files-5.15/arch/mips/rtl838x/prom.c

```diff
@@ -20,6 +20,7 @@
 #include <asm/cpu.h>
 #include <asm/smp-ops.h>
 #include <asm/mips-cps.h>
+#include <asm/fw/fw.h>
```

Comment:
keep your headers sorted; even if the rest isn't; so put it on 21 (before smp-ops)